addons/mass_mailing_crm/views/mass_mailing_views.xml

```diff
@@ -11,14 +11,16 @@
                     type="object"
                     icon="fa-star"
                     class="oe_stat_button"
-                    attrs="{'invisible': ['|', '|', ('state', 'in', ('draft','test')), ('mailing_model_name', '!=', 'crm.lead'), ('crm_lead_activated', '=', False)]}" >
+                    groups="sales_team.group_sale_salesman"
+                    attrs="{'invisible': ['|', ('state', '=', 'draft'), ('crm_lead_activated', '=', False)]}" >
```

Comment:
Shouldn't we put groups on the fields directly ?

Reply:
Good point, it's now done.